code/model/formfields/EditableLiteralField.php

```diff
@@ -12,14 +12,51 @@ class EditableLiteralField extends EditableFormField {
 	private static $singular_name = 'HTML Block';
 	
 	private static $plural_name = 'HTML Blocks';
+
+	/**
+	 * Safely sanitise html content, if enabled
+	 *
+	 * @param string $content Raw html
+	 * @return string Safely sanitised html
+	 */
+	protected function sanitiseContent($content) {
+		// Check if sanitisation is enabled
+		if(!HtmlEditorField::config()->sanitise_server_side) return $content;
+
+		// Perform sanitisation
+		$htmlValue = Injector::inst()->create('HTMLValue', $content);
+		$santiser = Injector::inst()->create('HtmlEditorSanitiser', HtmlEditorConfig::get_active());
```

Comment:
Hmm, more coupling with core, but I don't see a cleaner way. Maybe make the `HtmlEditorConfig` instance used configurable? In case you might want to allow more lenient processing in user forms, but stricter one in rich text editing? Might be overthinking it a bit though... your call.